server/router/rider.ts

```diff
@@ -1,120 +1,105 @@
 import express from 'express';
 import uuid from 'uuid/v1';
-import AWS from 'aws-sdk';
-import * as t from 'io-ts';
-import { isRight } from 'fp-ts/lib/Either';
-import config from '../config';
-import { deleteByID } from './common';
+import dynamoose from 'dynamoose';
+import { Locations } from './location';
 
 const router = express.Router();
 
-AWS.config.update(config);
-const docClient = new AWS.DynamoDB.DocumentClient();
-
 type Key = {
   id: string
 };
 
-const Body = t.type({
-  firstName: t.string,
-  lastName: t.string,
-  phoneNumber: t.string,
-  email: t.string,
-  accessibilityNeeds: t.type({
-    needsWheelchair: t.boolean,
-    hasCrutches: t.boolean,
-    needsAssistant: t.boolean,
-  }),
-  description: t.string,
-  joinDate: t.string,
-  pronouns: t.string,
-  address: t.string,
+type RiderType = {
+  id: string,
+  firstName: string,
+  lastName: string,
+  phoneNumber: string,
+  email: string,
+  accessibilityNeeds: {
+    needsWheelchair: boolean,
+    hasCrutches: boolean,
+    needsAssistant: boolean,
+  },
+  description: string,
+  joinDate: string,
+  pronouns: string,
+  address: string,
+  favoriteLocations: string[]
+};
+
+const schema = new dynamoose.Schema({
+  id: String,
+  firstName: String,
+  lastName: String,
+  phoneNumber: String,
+  email: String,
+  accessibilityNeeds: {
+    type: Object,
+    schema: {
+      needsWheelchair: Boolean,
+      hasCrutches: Boolean,
+      needsAssistant: Boolean,
+    },
+  },
+  description: String,
+  joinDate: String,
+  pronouns: String,
+  address: String,
+  favoriteLocations: {
+    type: Array,
+    schema: [{ type: String }],
+  },
 });
 
+export const Riders = dynamoose.model('Riders', schema, { create: false });
+
 // Get a rider by ID in Riders table
 router.get('/:id', (req, res) => {
   const { id } = req.params;
-  const params = {
-    TableName: 'Riders',
-    Key: { id },
-  };
-  docClient.get(params, (err, data) => {
+  Riders.get(id, (err, data) => {
     if (err) {
-      res.send({ err });
+      res.send(err);
+    } else if (!data) {
+      res.send({ err: { message: 'id not found' } });
     } else {
-      res.send(data.Item);
+      res.send(data);
     }
   });
 });
 
 // Get all riders
 router.get('/', (req, res) => {
-  const params = {
-    TableName: 'Riders',
-  };
-  docClient.scan(params, (err, data) => {
+  Riders.scan().exec((err, data) => {
     if (err) {
-      res.send({ err });
+      res.send(err);
+    } else if (!data) {
+      res.send({ err: { message: 'items not found' } });
     } else {
-      res.send({ data: data.Items });
+      res.send(data);
     }
   });
 });
 
-// Get all upcoming rides for a rider
+// TODO: Get all upcoming rides for a rider
 router.get('/:id/rides', (req, res) => {
```

Comment:
Could you possibly re-implement this in this PR. Otherwise, leave it as is. Do not remove the functionality in this PR if it will be changed in another.